Source/BugsnagKSCrashSysInfoParser.m

```diff
@@ -10,77 +10,20 @@
 #import "Bugsnag.h"
 #import "BugsnagCollections.h"
 #import "BugsnagKeys.h"
-#import "BugsnagConfiguration.h"
 #import "Private.h"
```

Comment:
I think `Bugsnag.h`, `BugsnagKeys.h` and `proivate.h` can be removed here.

Reply:
Moved the function definition to `BugsnagDeviceWithState.m` to avoid ballooning `BugsnagEvent`, and added a forward declaration where it is used in the event class.